modules/shortcodes/dailymotion.php

```diff
@@ -110,12 +131,12 @@ function dailymotion_shortcode( $atts ) {
 	}
 
 	/*set width and height using provided parameters if any */
-	$width = isset($atts['width']) ? abs( $atts['width'] ) : 0 ;
-	$height = isset($atts['height']) ? abs( $atts['height'] ) : 0 ;
+	$width  = isset( $atts['width'] )  ? intval( $atts['width'] )  : 0 ;
+	$height = isset( $atts['height'] ) ? intval( $atts['height'] ) : 0 ;
 
-	if ( ! $width && ! $height) {
+	if ( ! $width && ! $height ) {
 		if ( ! empty( $content_width ) ) {
-			$width = min( 425, intval( $content_width ) );
+			$width = absint( $content_width );
```

Comment:
hi, I just have a comment here, the old code used to have `$width = min( 425, intval( $content_width ) );`, this is why I left if, for compatibility reasons. I'm not against using the value when set, but aren't we risking to break the existing usages of this shortcode if we change this?